tesseract/tesseract_collision/test/collision_box_capsule_unit.cpp

```diff
@@ -0,0 +1,165 @@
+#include <tesseract_common/macros.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <gtest/gtest.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_POP
+
+#include "tesseract_collision/bullet/bullet_discrete_simple_manager.h"
+#include "tesseract_collision/bullet/bullet_discrete_bvh_manager.h"
+#include "tesseract_collision/fcl/fcl_discrete_managers.h"
+
+using namespace tesseract_collision;
+using namespace tesseract_geometry;
+
+void addCollisionObjects(DiscreteContactManager& checker)
+{
+  //////////////////////
+  // Add box to checker
+  //////////////////////
+  CollisionShapePtr box(new Box(1, 1, 1));
+  Eigen::Isometry3d box_pose;
+  box_pose.setIdentity();
+
+  CollisionShapesConst obj1_shapes;
+  tesseract_common::VectorIsometry3d obj1_poses;
+  obj1_shapes.push_back(box);
+  obj1_poses.push_back(box_pose);
+
+  checker.addCollisionObject("box_link", 0, obj1_shapes, obj1_poses);
+
+  /////////////////////////////////////////////
+  // Add thin box to checker which is disabled
+  /////////////////////////////////////////////
+  CollisionShapePtr thin_box(new Box(0.1, 1, 1));
+  Eigen::Isometry3d thin_box_pose;
+  thin_box_pose.setIdentity();
+
+  CollisionShapesConst obj2_shapes;
+  tesseract_common::VectorIsometry3d obj2_poses;
+  obj2_shapes.push_back(thin_box);
+  obj2_poses.push_back(thin_box_pose);
+
+  checker.addCollisionObject("thin_box_link", 0, obj2_shapes, obj2_poses, false);
+
+  /////////////////////////////////////////////////////////////////
+  // Add capsule to checker. If use_convex_mesh = true then this
```

Comment:
Should this be tested with use_convex_mesh = true as well? I don't see that anywhere.

Reply:
I will remove the comment